src/helpers.js

```diff
@@ -84,3 +84,42 @@ export const getMergedOptions = customOptions => {
 export const mergeColors = customColors => {
     return {...defaultColors, ...customColors};
 };
+
+//////////////// STORYBOOK EXAMPLES //////////////// 
```

Comment:
Everything after here should be put into a file in the `.storybook` directory so they don't end up in the output build.

In fact, we might want to move the stories outside of `src` as well, and make `examples` a root-level directory to distinguish between what *is* and *isn't* a part of the library.

Reply:
Yeah, was thinking I'd avoid that until the next PR. Didn't wana make too big of a change.